packages/components/molecules/f-promotions-showcase/src/components/PromotionsShowcase.vue

```diff
@@ -1,32 +1,224 @@
+        },
+
+        isFunction,
+
+        isVueComponent (component) {
+            return component?.render instanceof Function
+                || isString(component?.template)
+                || (component instanceof Function && component.name === 'VueComponent');
+        },
+
+        itemTag (item) {
+            if (isString(item.link) || isString(item.link?.href)) {
+                return 'a';
+            }
+
+            return isFunction(item.link)
+                ? 'button'
+                : 'div';
+        },
+
+        linkAttrs (item) {
+            if (isFunction(item?.link)) {
+                return false;
+            }
+
+            return isString(item?.link)
+                ? { href: item?.link }
+                : { ...item?.link };
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
 
+$promotionsItems-borderRadius                        : $radius-rounded-c;
+
 .c-promotionsShowcase {
+    background-color: $color-support-brand-02;
+    padding: spacing(x2);
+}
+
+.c-promotionsShowcase--inner {
     display: flex;
+    flex-direction: column;
     justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid $color-red;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
+    background-color: $color-container-default;
+
+    box-shadow: 0 4px 6px       rgba(27, 35, 36, 0.02),
+                0 2px 12px -2px rgba(27, 35, 36, 0.08),
+                0 3px 6px       rgba(27, 35, 36, 0.06);
+
+    border-radius: $promotionsItems-borderRadius;
+
+    @include media ('>=wide') {
+        flex-direction: row;
+    }
+}
+
+.c-promotionsShowcase--item {
+    display: flex;
+    flex-direction: row;
+    padding: spacing(x2);
+    text-decoration: none;
+
+    // Override button styles
+    border: none;
+    background: transparent;
+    font-family: inherit;
+    text-align: inherit;
+
+    @include media ('>=wide') {
+        flex-grow: 1;
+    }
+
+    &:not(*:first-child) {
+        border-top: 1px solid $color-border-default;
+
+        @include media ('>=wide') {
+            border-top: none;
+        }
+    }
+}
+
+.c-promotionsShowcase--itemElement {
+    &:not(*:first-child) {
+        margin-left: spacing(x2);
+    }
+}
+
+.c-promotionsShowcase--itemElement--styleText,
+.c-promotionsShowcase--itemElement--styleLink,
+.c-promotionsShowcase--itemElement--styleEmphasized {
+    &:nth-of-type(1) {
+        margin-top: spacing(x0.5);
+    }
+
+    margin-top: spacing();
+}
+
+.c-promotionsShowcase--itemInteractive {
+    cursor: pointer;
+}
+
+.c-promotionsShowcase--itemElement--styleLink {
+    color: $color-content-link;
+    font-weight: $font-weight-bold;
+
+    .c-promotionsShowcase--item:hover & {
+        text-decoration: underline;
+    }
+}
+
+
+.c-promotionsShowcase--itemElement--styleEmphasized {
+    text-decoration: underline;
+    font-weight: $font-weight-bold;
+}
+
+.c-promotionsShowcase--itemIllustrationContainer {
```

Comment:
Should use a single dash here to show it's a child element of `c-promotionsShowcase`. The double dashes are reserved for modifiers.

```suggestion
.c-promotionsShowcase-itemIllustrationContainer {
```